data/content/agent_attributes.yaml

```diff
@@ -4,13 +4,13 @@ attributes:
   default_value: None
   required: true
   desc: |
-      The agent registration token from your organization's _Agents_ page, used only to register new agents. To get your token, log in to Buildkite, navigate to _Agents_, then select _Reveal Agent Token_.
+      The agent token associated with a cluster (and configurable through the cluster accessible from your organization's **Agents** page).
 - name: build-path
   env_var: BUILDKITE_BUILD_PATH
   default_value: "(depends on platform)"
   required: true
   desc: |
-      Path to where the builds will run from.
+      The path from where the builds will be run.
```

Comment:
Ah interesting point, I was thinking of 'path' as a synonymous term with directory, which is more or less true in this case, but the path does inherently incorporate not just the directory, but all other ancestral directories to the root too.
I was also trying to avoid ending a sentence with a preposition, something that was ingrained during my school years but is gradually being 💩 'ed on.

Anyway, I'll change this to:

> The path to the directory where builds will be run from.

Reply:
Oh! Ha. I forgot about that old chestnut! (the 'don't end a sentence with a preposition' rule)

Your updated change (or something you manage to improve from there!) sounds good to me.